### ISSUE TITLE
"New"-label for new products

### ISSUE THREAD

**Issue by krestenlaust** [f-klubben/stregsystemet#442]
Having a date range within which there will be shown a "New" label showing that a new product code has been introduced would be cool. It would serve as a way to guide fembers into choosing the right product-code, e.g. when a Beer of the month becomes part of the fixed selection

Simulation:
![Screenshot of stregsystem with 'new' put on to of some wares](https://github.com/f-klubben/stregsystemet/assets/17174333/e173572e-a916-4a3b-8d38-14c8c723d710)

**Comment by Zeps2369** [comment]
Would a fixed date for every product be the best solution or being able to set a range when a new product is added?

**Comment by krestenlaust** [comment]
I think the former. Because for some products you probably wouldn't want to have the new tag, if it's just a technicality. Also, sometimes you'd want to add. A new tag to an existing product, e.g. månedens øl

**Comment by Zeps2369** [comment]
yeah i've been looking at the models right now, and i think a simple date value for when new is to be expired would be a good solution

**Comment by krestenlaust** [comment]
Good idea!
